plugins/csv-parser/src/dataParser.ts

```diff
@@ -18,9 +22,14 @@ export interface csvTypes {
 	options?: optionsTypes
 }
 
-export type hookContexts = {
-	target: Anim.AnimTarget & { data: { csv?: csvTypes } }
-	options?: Anim.ControlOptions
+export type AnimTarget = PrepareAnimationContext | { data: { csv?: csvTypes } }
```

Comment:
This does not seem right. the shape of the PrepareAnimationContext is: 
```
{
  target: [
    {
       target: {
         data: { ... },
         ...
       },
       ...
     }
   ]
}
 ```

also, I'm not sure intersection type works here, since csv can be specified alongside with other configuration.

Reply:
resolved